docs/src/content/docs/uk/getting-started.mdx

```diff
@@ -0,0 +1,174 @@
+
+Щоби запустити розробницький сервер, виконайте таку команду в теці вашого проєкту:
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+npm run dev
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+pnpm dev
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+yarn dev
+```
+
+</TabItem>
+</Tabs>
+
+У термінал видрукується URL локального сервера.
+Відкрийте цю ланку, щоби переглянути сайт.
+
+### Додайте контент
+
+Starlight підготовлено, тепер ви можете додавати новий контент або долучати наявні файли!
+
+#### Формати файлів
+
+Starlight підтримує створення контенту у форматах Markdown і MDX без жодних додаткових налаштувань.
+Ви можете додати підтримку Markdoc, встановивши експериментальну [інтеґрацію Astro Markdoc](https://docs.astro.build/guides/integrations-guide/markdoc/).
+
+#### Додайте сторінки
+
+Додавайте нові сторінки на ваш сайт, створюючи файли `.md` та `.mdx` у теці `src/content/docs/`.
+Використовуйте підтеки, щоб орґанізовувати файли та створювати множинні сеґменти шляху.
+
+Наприклад, оттака структура файлів зґенерує сторінки за адресами `example.com/hello-world` і `example.com/guides/faq`:
+
+import FileTree from '../../../components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - faq.md
+      - hello-world.md
+
+</FileTree>
+
+#### Безпека типів у преамбулі (frontmatter)
+
+У Starlight усі сторінки поділяють спільний налаштовуваний [набір властивостей преамбули](/uk/reference/frontmatter/), якими визначається вигляд сторінки:
+
+```md
+---
+title: Привіт, світ!
+description: Це сторінка на моїм сайті, що він працює на Starlight!
+---
+```
+
+Якщо ви забудете щось важливе, Starlight сповістить вас про це.
+
+### Наступні кроки
+
+- **Конфіґурація:** Дізнайтеся про поширені параметри в ґайді [“Налаштовування Starlight”](/uk/guides/customization/).
+- **Навіґація:** Спорудить бічну панель у розділі [“Навіґація бічною панеллю”](/uk/guides/sidebar/).
+- **Компоненти:** Відкрийте вбудовані картки, вкладки та багато іншого в [“Компонентах”](/uk/guides/components/).
+- **Розгортування:** Опублікуйте власну працю за допомогою посібника [“Розгорніть свій сайт”](https://docs.astro.build/guides/deploy/) в документації Astro.
+
+## Оновлення Starlight
+
+:::tip
+Оскільки Starlight поки на стадії beta, очікуйте частих оновлень.
+Не забувайте регулярно оновлювати Starlight!
+:::
+
+Starlight є інтеґрацією Astro, отже ви можете оновити його так само, як і решту пакетів Astro, ─ виконавши в терміналі команду:
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+npx @astrojs/upgrade
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+pnpm dlx @astrojs/upgrade
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+yarn dlx @astrojs/upgrade
+```
+
+</TabItem>
+</Tabs>
+
+Повний список змін до кожного релізу дивіться в [списку змін Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md).
+
+## Зневадження Starlight
```

Comment:
Just want to get a double check on this one

Reply:
@juchym would this make sense for “troubleshooting”?

```suggestion
## Усунення несправностей Starlight
```

Reply:
Hello, I used [this dictionary](https://e2u.org.ua/s?w=troubleshoot*&highlight=on) to translate the word. To me, their meaning is close enough, I have no preference here. I chose "зневадження", because it looked more concise. Also, possible two word translations may include
* Усунення/вилучення вад/пошкод/негараздів/помилок (literally 'eliminating flaws/errors')
* Вирішення проблем (as in 'resolving problems')
* Лагодження/вилагодження (more like 'fixing')

Reply:
Thanks! If you’re happy with the current version, then I am too — just wanted to double check given the machine translation we used wasn’t working for this one.